wagtail/wagtail
# Fix/remove defunct oembed providers

## Description

This PR removes several defunct oEmbed providers that no longer exist or have shut down their services. This fixes issues where Wagtail attempts to connect to non-existent services, which can cause unnecessary delays or connection timeouts.

### Removed Providers:
- app.net (shut down in 2014)
- bambuser (discontinued)
- blip.tv (shut down in 2015)
- qik (no longer exists)
- rdio (shut down in 2015)
- revision3 (no longer exists)
- vhx.tv (acquired and service changed)

## Motivation and Context

Fixes #12054 

Currently, Wagtail's oEmbed provider list includes several services that no longer exist. This causes issues when:
1. Attempting to embed content from these defunct services (returns `EmbedNotFoundException`)
2. Some providers that aren't accepting connections cause the request to hang (e.g., qik.com)

## How to Test

1. Verify that removing these providers doesn't affect existing functionality:
```python
from wagtail.embeds.embeds import get_embed
# Test with an active provider
get_embed('https://www.youtube.com/watch?v=example')
```
Thank you for the thorough explanation of the problem statement. If I have misinterpreted anything, I am open to any necessary changes. I hope this will help resolve the issue.
